lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -12,5 +12,41 @@ def fulfillments(options = {})
       fo_fulfillments = get(:fulfillments, options)
       fo_fulfillments.map { |fof| FulfillmentOrderFulfillment.new(fof.as_json) }
     end
+
+    def move(new_location_id:)
+      body = { fulfillment_order: { new_location_id: new_location_id } }.to_json
+      keyed_fos = load_keyed_attributes_from_response(post(:move, {}, body))
```

Comment:
I think the naming in this method could be simplified to improve readability.

you call the method `load_keyed_attributes_from_response` but if I'm following the logic correctly I think if there were actual `fulfillment order response` attributes for one of the following keys :
```
 original_fulfillment_order: 
 moved_fulfillment_order: 
 remaining_fulfillment_order:
```

you will load the actual `fulfillmentOrder` so the method returns `keyed_fulfillment_orders` and that's why the variable is named `keyed_fos` but the method name doesn't convey this information.

thoughts?

Reply:
👍, `load_keyed_fulfillment_orders_from_response`?